libraries/scripting/dependencies/src/kotlin/script/experimental/dependencies/annotations.kt

```diff
@@ -18,7 +16,7 @@ import kotlin.script.experimental.api.makeFailureResult
 @Target(AnnotationTarget.FILE)
 @Repeatable
 @Retention(AnnotationRetention.SOURCE)
-annotation class DependsOn(vararg val artifactsCoordinates: String)
+annotation class DependsOn(vararg val artifactsCoordinates: String, val options: String = "")
```

Comment:
I sill think that having `val options: Array<String> = []` is better here. It will leave less room for misinterpretation (which separators to use, how to escape strings with spaces, etc.) and should make the parser simpler.

Reply:
My mistake ✅